### ISSUE TITLE
Setup AWS VPN for database access

### ISSUE THREAD

**Issue by patheard** [cds-snc/notification-planning#613]
## Description

As a developer,
I want a secure way to access the Notify database instance locally,
so that I can troubleshoot issues Notify data without worrying about unauthorized access to the data.

## Acceptance Criteria** (Definition of done) 

- [ ] Setup [AWS VPN for Staging and Production accounts](https://docs.google.com/document/d/1uFZ2josUt9-3fbohwauR8n2ACzel5wNqHtEgN8xepvE/edit#heading=h.8y8kbj277q9j).
- [ ] [Configure VPN access for each team member](https://aws.amazon.com/premiumsupport/knowledge-center/client-vpn-multiple-users-same-endpoint/) that requires access to the database.
- [ ] Remove the jumpbox and hasura pods.

## QA Steps

- [ ] Connect to the Staging and Production VPN using your credentials.
- [ ] Test that you can access the database while connected to the VPN.
- [ ] Test that you can access Redis while connected to the VPN.
- [ ] Check that the jumpbox and hasura pods are no longer deployed to Kubernetes.

**Comment by jimleroyer** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/613?planning-poker) with ZenHub @patheard @sastels

**Comment by patheard** [comment]
I've also used it for Redis access, but that could be setup with the VPN as well.

**Comment by jimleroyer** [comment]
VPN is installed from within the network, but restrict the access? We need to allow extra usage, unlike the jumpbox? Just wondering. (if we can still access Redis, that would certainly be helpful!)

**Comment by patheard** [comment]
Yup, the way you set up the VPN is to grant it access to specific security groups, so in the case of Redis, we'd just need to allow the VPN traffic access to the EKS cluster security group:
https://github.com/cds-snc/notification-terraform/blob/main/aws/elasticache/elasticache.tf#L24-L26
